src/arm/BBORG_SERVO-00A2.dts

```diff
@@ -35,27 +35,27 @@
 
     P9_19_pinmux { status = "disabled"; };  /* i2c2.scl */
     P9_20_pinmux { status = "disabled"; };	/* i2c2.sda */
-    P8_10_pinmux { status = "disabled"; };	/* servo.enable */
+    P8_10_pinmux { status = "disabled"; };	/* pca9685.enable */
 };
 
 &am33xx_pinmux {
 
     bborg_servo_input_pins: pinmux_bborg_servo_input_pins {
         pinctrl-single,pins = <
-            BONE_P8_12 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P8_12: gpio1_12 */
-            BONE_P8_11 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P8_11: gpio1_13 */
-            BONE_P9_30 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P9_30: gpio3_16 */
-            BONE_P9_27 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P9_27: gpio3_19 */
-            BONE_P9_41 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P9_41: gpio3_20 */
-            BONE_P9_42 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P9_42: gpio0_7 */
+            BONE_P8_12 (PIN_INPUT_PULLDOWN | MUX_MODE7)    /* P8_12: gpio1_12 */
```

Comment:
What are these pins for? I thought they were to configure the device. Shouldn't they be set to output GPIOs from the board and then set to properly configure the device?

Reply:
these are the RC input pins.

Reply:
But what do they do? I know I should know, but it really isn't clear to me. At first, I thought these were PWM inputs, but looking at the datasheet it seems these are just configuration pins.

Reply:
They take input signal from a remote control, you shared a link on the group which shows different types of signals we may encounter from the receiver https://oscarliang.com/pwm-ppm-sbus-dsm2-dsmx-sumd-difference/. I asked others on the group and we agreed to have GPIO inputs here.

I think the main purpose is to take input from those RC receivers and control servos on a drone, car, plane, etc.

Reply:
This is a flat-out misinterpretation and bad guess. Please check the datasheet https://www.nxp.com/docs/en/data-sheet/PCA9685.pdf.